src/UserProfileAndSettingsPage/PersonalDetails.tsx

```diff
@@ -79,10 +79,10 @@ class PersonalDetails extends React.Component<
           <div className="settings-labels">
             <label className="updateUserLabel">UserId</label>
             <input
-              className="settings-input"
+              className="settings-input unavailible"
               name="userId"
               value={this.state.userId}
```

Comment:
you still havent changed userId. should be username now right? gotta change the label, name, and value